shuttheboxgame.py

```diff
@@ -46,21 +51,47 @@ def shutthebox_v1(tiles):
         old_tiles = tiles.copy()
         strat.__startlow__()
 
+        # Check to see if all tiles have been knocked down
+        if not tiles:
+            score = 0
+            return f"The game is over! Your victory took {num_rolls} rolls! Congratulations!", score
+
         #Check to see if no tiles were able to be knocked down
         if old_tiles == tiles:
-            return print(f"No more tiles can be knocked down. Here is your final score: {''.join(map(str,tiles))}.")
+            score = ''.join(map(str,tiles))
```

Comment:
Any benefit to end with the score as a string? Feels like a int would be more valuable

Reply:
nah, was just doing that to concatenate the numbers. I can convert it back to int in that line though. Unless you know a better way